test/unit/spec/appWidgets/validationSpec.js

```diff
@@ -6,7 +6,31 @@ define([
 ], (Promise, Validation, Constants, TestUtil) => {
     'use strict';
 
+    const aLoadOfInputs = [
+        null,
+        undefined,
+        '',
+        'string',
+        42,
+        [],
+        {},
+        [1, 2, 3],
+        { this: 'that' },
+        () => {
+            return;
+        },
+    ];
     describe('Validator functions', () => {
+        it('Is alive', () => {
+            let alive;
```

Comment:
This could really just be `expect(Validation).toBeTruthy()`. Or, more specifically, `jasmine.any(Object)`

Reply:
Oh, I see below, it was just moved without modification. That's fine.